TASVideos/Pages/Publications/Models/PublicationRateModel.cs

```diff
@@ -1,17 +1,62 @@
-﻿using System.ComponentModel.DataAnnotations;
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Globalization;
 
 namespace TASVideos.Pages.Publications.Models
 {
 	public class PublicationRateModel
 	{
+		public sealed class RatingString : ValidationAttribute
```

Comment:
I've never seen a validation attribute slammed into a page model before.  I guess if the intended scope is ONLY this page, that's fine.  But we also have a folder for these things